### PR TITLE
Expose tts volume and pitch to the user code like we do with lang.

### PR DESCRIPTION
**Pull request by divx118**
Like the title says, this will expose the volume and pitch for tts to the user code. 
It can be set/get globally with:
```
aiy.audio.set_tts_volume(50)
volume = aiy.audio.get_tts_volume()
aiy.audio.set_tts_pitch(140)
pitch = aiy.audio.get_tts_pitch()
```
Or set individually with say:
```
aiy.audio.say('Good bye!', volume=50, pitch=140)
```

### CODE REVIEW

File: src/aiy/audio.py [hunk 1]
```diff
@@ -108,15 +110,26 @@ def play_audio(audio_data):
...
 
-    If lang is specified, e.g. "en-US', it will be used to say the given words.
+    If lang is specified, e.g. "en-US", it will be used to say the given words.
     Otherwise, the language from aiy.i18n will be used.
+    volume (optional) volume used to say the given words.
+    pitch (optional) pitch to say the given words.
+    Example: aiy.audio.say('This is an example', lang="en-US", volume=75, pitch=135)
+    Any of the optional variables can be left out.
     """
+    global _tts_volume
```
Comment (drigz) [inline src/aiy/audio.py]: Remove - these aren't required in this function, as it uses get_*_volume.

File: src/aiy/_drivers/_tts.py [hunk 2]
```diff
@@ -20,6 +20,7 @@
 import subprocess
 import tempfile
 from aiy import i18n
+from aiy import audio
```
Comment (drigz) [inline src/aiy/_drivers/_tts.py]: Remove - this creates a circular import between the two modules, which could lead to confusing problems later.

File: src/aiy/_drivers/_tts.py [hunk 3]
```diff
@@ -30,24 +31,29 @@
 def create_say(player):
     """Return a function say(words) for the given player."""
     lang = i18n.get_language_code()
-    return functools.partial(say, player, lang=lang)
+    volume = audio.get_tts_volume()
+    pitch = audio.get_tts_pitch()
+    return functools.partial(say, player, lang=lang, volume=volume, pitch=pitch)
```
Comment (drigz) [inline src/aiy/_drivers/_tts.py]: I'd suggest ignoring volume and pitch here and letting `say()` handle it.

File: src/aiy/audio.py (hunk 1 above)
Comment (divx118) [reply src/aiy/audio.py]: Yeah sorry about that. Didn't do much python programming yet.

**Comment by drigz** [comment]
Merged - works like a charm! Thank you for contributing.

**Comment by divx118** [comment]
No problem. This benefits me too :).

**Pull request merged**